lib/mediawiki/query/lists/search.rb

```diff
@@ -36,6 +36,24 @@ def get_search_results(search_value, namespace = 0)
           query_ary(params, 'search', 'title')
         end
 
+        # Gets an array containing page titles that matched the search.
+        # @param search_value [String] The thing to search for.
+        # @param namespace [Fixnum] The namespace to search in. Defaults to 0 (the main namespace).
+        # @see https://www.mediawiki.org/wiki/API:Search MediaWiki Search API Docs
+        # @since 3.0.1
```

Comment:
It's more likely this would be 3.1.0, but this should be removed anyway because it isn't decided what version to include this change in.